src/main/java/carly/utils/TaskList.java

```diff
@@ -0,0 +1,240 @@
+package carly.utils;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+import carly.exception.CarlyException;
+import carly.exception.CarlyIncorrectIndexFormat;
+import carly.exception.CarlyIndexOutOfBoundsException;
+import carly.exception.CarlyMissingDateTimeException;
+import carly.tasks.Deadline;
+import carly.tasks.Event;
+import carly.tasks.Task;
+import carly.tasks.Todo;
+import carly.ui.Ui;
+
+/** Represents a list of tasks. Handle a collection of {@link Task} objects. */
+public class TaskList {
+
+    /** List of Task objects. */
+    private final ArrayList<Task> taskList;
+
+    /** TWO_INDENT used in displaying tasklist size. */
+    private final String ONE_INDENT = "    ";
```

Comment:
Adheres to good coding standards by writing constant variables in SCREAMING_SNAKE_CASE